### ISSUE TITLE
How to scope log to target?

### ISSUE THREAD

**Issue by lolgear** [CocoaLumberjack/CocoaLumberjack#887]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [x] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / osx / tvos / watchos
 Platform Version        | e.g. 8.0
 CocoaLumberjack Version | e.g. 2.3.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

It is a how-to question, excuse me.

Setup:
Several targets, three frameworks and application target.

How to disable log in specific framework?

I could imagine something like:

```
#define CUSTOM_FRAMEWORK_DDLogDebug(...) if (LOG_ENABLED_FOR_CUSTOM_FRAMEWORK) {DDLogDebug(...)} 
```

This is a simple flag-based solution. Flag could be retrieved from environment or somewhere else.

Is there any ready-to-use solution or some hints in this task?

**Comment by bpoplauschi** [comment]
@lolgear I think the answer you also gave is the only one :) I don't know about another way to achive this
